src/main/java/github/scarsz/discordsrv/listeners/DiscordBanListener.java

```diff
@@ -58,7 +59,7 @@ public void onGuildBan(GuildBanEvent event) {
         if (offlinePlayer.isOnline()) {
             // also kick them because adding them to the BanList isn't enough
             Player player = offlinePlayer.getPlayer();
-            if (player != null) Bukkit.getScheduler().runTask(DiscordSRV.getPlugin(), () -> player.kickPlayer(reason));
+            if (player != null) SchedulerUtil.runTask(DiscordSRV.getPlugin(), () -> player.kickPlayer(reason));
```

Comment:
This needs to use the new method as well

Reply:
Done. Don't hesitate to ping me if I've forgotten something or if tests fails.